tests/dummy/app/templates/application.hbs

```diff
@@ -0,0 +1,5 @@
+{{!-- The following component displays Ember's default welcome message. --}}
+{{welcome-page}}
```

Comment:
Is this file actually needed?
Would this even work without the ember-welcome-page package?
https://github.com/ember-cli/ember-addon-output/blob/master/tests/dummy/app/templates/application.hbs doesn't have this.